x/slashing/keeper.go

```diff
@@ -40,7 +41,7 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger { return ctx.Logger().With("m
 
 // handle a validator signing two blocks at the same height
 // power: power of the double-signing validator at the height of infraction
-func (k Keeper) handleDoubleSign(ctx sdk.Context, addr crypto.Address, infractionHeight int64, timestamp time.Time, power int64) {
+func (k Keeper) HandleDoubleSign(ctx sdk.Context, addr crypto.Address, infractionHeight int64, timestamp time.Time, power int64) {
```

Comment:
I noticed a lot of changes going from private to public visibility? 
Are these always necessary?

Reply:
Yes they are because they are required to be used by the handlers. These changes are not a security risk

Reply:
Private vs public is not about security is about encapsulation.
Making many things public affects the architecture and the public API of the module.
We should be very conservative when moving things from private to public, specially if it is just for comfort :)

Reply:
sure I agree - this is required here though.